ffmpeg/ffmpeg_test.go

```diff
@@ -804,6 +804,74 @@ func TestTranscoder_StreamCopy(t *testing.T) {
 	}
 }
 
+func TestTranscoder_StreamCopy_Validate_B_Frames(t *testing.T) {
+	run, dir := setupTest(t)
+	defer os.RemoveAll(dir)
+
+	// Set up inputs, truncate test file
+	cmd := `
+        set -eux
+        cd "$0"
+        cp "$1"/../transcoder/test.ts .
+        ffmpeg -i test.ts -c:a copy -c:v copy -t 1 test-short.ts
+
+        # sanity check some assumptions here for the following set of tests
+        ffprobe -count_frames -show_streams -select_streams v test-short.ts | grep nb_read_frames=60
+    `
+	run(cmd)
+
+	// Test normal stream-copy case
+	in := &TranscodeOptionsIn{Fname: dir + "/test-short.ts"}
+	out := []TranscodeOptions{
+		TranscodeOptions{
+			Oname:        dir + "/audiocopy.ts",
+			Profile:      P144p30fps16x9,
+			AudioEncoder: ComponentOptions{Name: "copy"},
+		},
+		TranscodeOptions{
+			Oname: dir + "/videocopy.ts",
+			VideoEncoder: ComponentOptions{Name: "copy", Opts: map[string]string{
+				"mpegts_flags": "resend_headers,initial_discontinuity",
```

Comment:
Just realized we also have this option enabled in `TestTranscoder_StreamCopy` and I have absolutely no recollection of why it's there [1]. Do your tests still pass without the option? If so, we can leave the `Opts` unset here for clarity. (Things pass here without it.)

[1] it *might* have been some ffmpeg-specific weirdness that has since been fixed in master

Reply:
Works even without `Opts`. Removed Opts